### PR TITLE
reverting claim in test

### PR DESCRIPTION
**Pull request by davidbecker**
removed claim from #78

### CODE REVIEW

**Comment by tommyettinger** [comment]
I'm pretty sure that was a rare case where the LightRNG was actually generating 0 at that point in the sequence of pseudo-random numbers, and since 0 is always <= the bound in nextLong(long bound), it never needed to generate any further pseudo-randomness and 0 was always an acceptable value. (Each 64-bit value it generates causes the state to be raised by a large fixed value, which often overflows to make state negative again; it could have had a value exactly one step below 0, which might not be as unlikely as it seems, considering the initial state was 2, and that is only one bit-flip away from 0.)
I can't currently merge this since I found a bug in PermutedRandom and so several related things also needed to change, including a big one (LightRNG would never generate negative numbers with nextLong(), despite the docs saying that it could and any claims of high-quality pseudo-randomness depending on it). So that's one bug found by testing already, good job @davidbecker !

**Comment by davidbecker** [comment]
When I woke up this morning I figured that 0 might be "randomly" the next number in the chain and expanded the test.
I never thought that this test would produce meaningful results, or dig up any bugs, so "yeah, testing" :smile:

**Comment by davidbecker** [comment]
I just ran the test with a breakpoint on this:
nextLong() would return 9043571121738711001
bits = 4521785560869355500 (and since it has xx500 at the end ->)
value = 0
so everything was as expected. And it seems that I only choose 2, 5, 10 as a parameter at this point which doesn't make me a good random number generator, I guess...

**Comment by tommyettinger** [comment]
Glad to see that it wasn't a bug there, although the fact that LightRNG can now actually generate negative numbers with methods that couldn't before means the results all changed. I updated that stuff in a commit just now, and one flaw in the JUnit test: when comparing doubles or floats: a delta of `1 / 1000` is equivalent to a delta of `0` due to integer division acting as a floor. Since I assume you actually wanted one-one-thousandth, I changed those to `1.0 / 1000.0`. (This was relevant when I was handling the random floats; I am not sure how to get the values without truncating them and rounding off at the last number within supported precision for a float, and JUnit uses the entire many-digit representation without any rounding.) I'm closing this but I will incorporate the non-conflicting changes.

File: squidlib-util/src/test/java/squidpony/squidmath/LightRNGTest.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
  *
  */
 public class LightRNGTest {
+	private static final double DELTA = 0.00000000001;
```
Comment (davidbecker) [inline squidlib-util/src/test/java/squidpony/squidmath/LightRNGTest.java]: @tommyettinger you could use it this way (as a reference to your lates comment in #79

**Pull request closed without merging**